contrib/opencensus-ext-django/README.rst

```diff
@@ -40,6 +38,30 @@ for a complete reference.
             )''',
         }
     }
+    
+Additional configuration can be provided, please read
+`Customization <https://github.com/census-instrumentation/opencensus-python#customization>`_
+for a complete reference.
+
+For customization the exporter has to be instantiated separately
+
+.. code:: python
+
+    from opencensus.ext.azure.trace_exporter import AzureExporter
+    exporter = AzureExporter(service_name='mysite')
+    exporter.add_telemetry_processor(callback)
+
+    OPENCENSUS = {
+        'TRACE': {
+            'SAMPLER': 'opencensus.trace.samplers.ProbabilitySampler(rate=1)',
+            'EXPORTER': exporter
+        }
+    }
+    
+    def callback(envelope):
+    return False
+
+Both forms assumes Environmental Variable ``APPINSIGHTS_INSTRUMENTATIONKEY`` exists.
```

Comment:
I would remove this line because `opencensus-ext-django` shouldn't have anything to do with `azure`.

Reply:
Done